File: Python/test/wb6StatusTest.py
import sys
sys.path.append('../API')

import wb6Status

test_ip = "10.100.100.100";

wb6Sts = wb6Status.wb6Status( test_ip );

print "Version = " + wb6Sts.getVersion()
print "OWBus = " + wb6Sts.getOneWireBus()

print "Temp 0 = " + str(wb6Sts.getTemp( 0))
print "Temp 1 = " + str(wb6Sts.getTemp( 1))
print "Temp 2 = " + str(wb6Sts.getTemp( 2))
print "Temp 3 = " + str(wb6Sts.getTemp( 3))
print "Temp 4 = " + str(wb6Sts.getTemp( 4))

print "An Out 0 = " + str(wb6Sts.getAnOut( 0))
print "An Out 1 = " + str(wb6Sts.getAnOut( 1))
print "An Out 2 = " + str(wb6Sts.getAnOut( 2))
print "An Out 3 = " + str(wb6Sts.getAnOut( 3))

print "An In 0 = " + str(wb6Sts.getAnIn( 0))
print "An In 1 = " + str(wb6Sts.getAnIn( 1))
print "An In 2 = " + str(wb6Sts.getAnIn( 2))
print "An In 3 = " + str(wb6Sts.getAnIn( 3))

print "Dig In 0 = " + str(wb6Sts.getDigIn( 0))
print "Dig In 1 = " + str(wb6Sts.getDigIn( 1))
print "Dig In 2 = " + str(wb6Sts.getDigIn( 2))
print "Dig In 3 = " + str(wb6Sts.getDigIn( 3))
print "Dig In 4 = " + str(wb6Sts.getDigIn( 4))
print "Dig In 5 = " + str(wb6Sts.getDigIn( 5))
print "Dig In 6 = " + str(wb6Sts.getDigIn( 6))
print "Dig In 7 = " + str(wb6Sts.getDigIn( 7))

print "Monitor 0 = " + str(wb6Sts.getMonitor( 0))
print "Monitor 1 = " + str(wb6Sts.getMonitor( 1))
print "Monitor 2 = " + str(wb6Sts.getMonitor( 2))
print "Monitor 3 = " + str(wb6Sts.getMonitor( 3))

print "Dig Out 0 = " + str(wb6Sts.getDigOut( 0))
print "Dig Out 1 = " + str(wb6Sts.getDigOut( 1))
print "Dig Out 2 = " + str(wb6Sts.getDigOut( 2))
print "Dig Out 3 = " + str(wb6Sts.getDigOut( 3))
print "Dig Out 4 = " + str(wb6Sts.getDigOut( 4))
print "Dig Out 5 = " + str(wb6Sts.getDigOut( 5))
print "Dig Out 6 = " + str(wb6Sts.getDigOut( 6))
print "Dig Out 7 = " + str(wb6Sts.getDigOut( 7))

print "Last cmd error = " + str(wb6Sts.getCmdError())
print "Operational State = " + str(wb6Sts.getOperationalState())
print "LoginState = " + str(wb6Sts.getLoginState())
print "Date = " + str(wb6Sts.getDate())
print "Time = " + str(wb6Sts.getTime())
print "Day = " + str(wb6Sts.getDay())

File: PHP/test/wb6StatusTest.php
<html>
<head>
<title>Standard WebBrick Control</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
</head>
<body bgcolor="#00CC66">
<?php
include("../API/wb.php") ;
include("../API/wb6.php") ;
include("../API/wb6status.php") ;

$test_ip = "10.100.100.100";

echo "New<BR>";
$wb6Sts = new wb6Status();
echo "Load<BR>";
$wb6Sts->load( $test_ip );
echo "Display<BR>";

echo "OWBus = " .$wb6Sts->getOneWireBus() ."<BR>";

echo "Temp 0 = " .$wb6Sts->getTemp( 0) ."<BR>";
echo "Temp 1 = " .$wb6Sts->getTemp( 1) ."<BR>";
echo "Temp 2 = " .$wb6Sts->getTemp( 2) ."<BR>";
echo "Temp 3 = " .$wb6Sts->getTemp( 3) ."<BR>";
echo "Temp 4 = " .$wb6Sts->getTemp( 4) ."<BR>";

echo "An Out 0 = " .$wb6Sts->getAnOut( 0) ."<BR>";
echo "An Out 1 = " .$wb6Sts->getAnOut( 1) ."<BR>";
echo "An Out 2 = " .$wb6Sts->getAnOut( 2) ."<BR>";
echo "An Out 3 = " .$wb6Sts->getAnOut( 3) ."<BR>";

echo "An In 0 = " .$wb6Sts->getAnIn( 0) ."<BR>";
echo "An In 1 = " .$wb6Sts->getAnIn( 1) ."<BR>";
echo "An In 2 = " .$wb6Sts->getAnIn( 2) ."<BR>";
echo "An In 3 = " .$wb6Sts->getAnIn( 3) ."<BR>";

echo "Dig In 0 = " .$wb6Sts->getDigIn( 0) ."<BR>";
echo "Dig In 1 = " .$wb6Sts->getDigIn( 1) ."<BR>";
echo "Dig In 2 = " .$wb6Sts->getDigIn( 2) ."<BR>";
echo "Dig In 3 = " .$wb6Sts->getDigIn( 3) ."<BR>";
echo "Dig In 4 = " .$wb6Sts->getDigIn( 4) ."<BR>";
echo "Dig In 5 = " .$wb6Sts->getDigIn( 5) ."<BR>";
echo "Dig In 6 = " .$wb6Sts->getDigIn( 6) ."<BR>";
echo "Dig In 7 = " .$wb6Sts->getDigIn( 7) ."<BR>";

echo "Monitor 0 = " .$wb6Sts->getMonitor( 0) ."<BR>";
echo "Monitor 1 = " .$wb6Sts->getMonitor( 1) ."<BR>";
echo "Monitor 2 = " .$wb6Sts->getMonitor( 2) ."<BR>";
echo "Monitor 3 = " .$wb6Sts->getMonitor( 3) ."<BR>";

echo "Dig Out 0 = " .$wb6Sts->getDigOut( 0) ."<BR>";
echo "Dig Out 1 = " .$wb6Sts->getDigOut( 1) ."<BR>";
echo "Dig Out 2 = " .$wb6Sts->getDigOut( 2) ."<BR>";
echo "Dig Out 3 = " .$wb6Sts->getDigOut( 3) ."<BR>";
echo "Dig Out 4 = " .$wb6Sts->getDigOut( 4) ."<BR>";
echo "Dig Out 5 = " .$wb6Sts->getDigOut( 5) ."<BR>";
echo "Dig Out 6 = " .$wb6Sts->getDigOut( 6) ."<BR>";
echo "Dig Out 7 = " .$wb6Sts->getDigOut( 7) ."<BR>";

echo "Last cmd error = " .$wb6Sts->getCmdError() ."<BR>";
echo "Operational State = " .$wb6Sts->getOperationalState() ."<BR>";
echo "LoginState = " .$wb6Sts->getLoginState() ."<BR>";
echo "Date = " .$wb6Sts->getWbDate() ."<BR>";
echo "Time = " .$wb6Sts->getTime() ."<BR>";
echo "Day = " .$wb6Sts->getDay() ."<BR>";
?>

</body>
</html>
 